3rdparty/CMakeLists.txt

```diff
@@ -79,6 +80,9 @@ option(PNG_TESTS  "Build libpng tests" OFF)
 add_subdirectory(libpng libpng EXCLUDE_FROM_ALL)
 add_dependencies(png_static zlib) #CMake doesn't get it itself
 
+#openexr
+add_subdirectory(openexr openexr EXCLUDE_FROM_ALL)
+
```

Comment:
here you must configure your lib, add all the options, build static, etc. just before

Reply:
shouldn't I configure lib in source CMakeLists?

Reply:
no, you must set up all the CMake options for the subfolder before you include it.